project-base/src/Controller/Front/NewsletterController.php

```diff
@@ -79,6 +121,7 @@ private function createSubscriptionForm(): Form
         /** @var \Symfony\Component\Form\Form $form */
         $form = $this->createForm(SubscriptionFormType::class, null, [
             'action' => $this->generateUrl('front_newsletter_send'),
+            'compound' => true,
```

Comment:
Can you please explain to me this configuration? Why is it needed?

Reply:
this is unnecessary, it was an attempt - 27223429b561378717bd3b0702a3ff030822cb8d